### PR TITLE
Add cuvette to remote jobs

### PR DESCRIPTION
**Pull request by sumitkr2000**
Hey, @JasonDsouza212 I have added cuvette to remote jobs. Please review my pr and merge it. Thank you!

Issue no. #407

### CODE REVIEW

**Comment by JasonDsouza212** [comment]
@sumitkr2000, I have mentioned some changes. Please make those changes so that I can merge the pull request (PR).

**Comment by sumitkr2000** [comment]
What changes do I have to make?? I can not see any mentions!!

**Comment by sumitkr2000** [comment]
@JasonDsouza212 Bro what changes are to be made?

**Comment by JasonDsouza212** [comment]
@sumitkr2000 In package-lock.json the version of vite has been changed to 4.3.9 so change it back to 4.3.5

**Comment by sumitkr2000** [comment]
> @sumitkr2000 In package-lock.json the version of vite has been changed to 4.3.9 so change it back to 4.3.5

Okay, bro...It may have changed because of installing dependency!!! I am fixing it.

**Comment by sumitkr2000** [comment]
@JasonDsouza212 Now please check

**Review by JasonDsouza212 — APPROVED** [review]

**Review by JasonDsouza212 — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
         "husky": "^8.0.3",
         "jsdom": "^22.0.0",
         "prettier": "^2.8.8",
-        "vite": "^4.3.5",
+        "vite": "^4.3.9",
```
[the remark this answers is not in the document]
Comment (sumitkr2000) [reply package-lock.json]: > Don't update the package version , so revert it to the older version

Bro it was updated was someone else so I left it as it was. So I should change the version of vite to 4.3.5 in package.json also?

**Pull request merged**